OJP/OJP_Common.xsd

```diff
@@ -87,6 +87,11 @@
 			<xs:documentation>Reference to an Operator ([TMv6] company  providing public transport services.) </xs:documentation>
 		</xs:annotation>
 	</xs:element>
+	<xs:complexType name="OperatorRelStructure">
```

Comment:
I think it should be operatorRefs_RelStructure.

Reply:
done